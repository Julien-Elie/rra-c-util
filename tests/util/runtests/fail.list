fail
missing
hup
status
skip
pass
