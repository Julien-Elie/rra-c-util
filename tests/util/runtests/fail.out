
Running all tests listed in fail.list.  If any tests fail, run the failing
test program by hand to see more details.  The test program will have the
same name as the test set but with ".t" appended.

fail............FAILED 2-3, 5-8
missing.........MISSED 1, 6, 10, 12, 14, 18; FAILED 8, 16
hup.............dubious (killed by signal 1)
status..........dubious (exit status 1)
skip............ok (skipped 3 tests)
pass............ok

Failed Set                 Fail/Total (%) Skip Stat  Failing Tests
-------------------------- -------------- ---- ----  ------------------------
fail                          6/8     75%    1    0  2-3, 5-8
missing                       8/19    42%    0    0  1, 6, 8, 10, 12, 14, ...
hup                           0/4      0%    0   --  
status                        0/4      0%    0    1  

Failed 14/51 tests, 72.55% okay.
Files=6,  Tests=51
